tests/test_e2e_13_mef_eline.py

```diff
@@ -0,0 +1,1117 @@
+            },
+            "uni_z": {
+                "interface_id": "00:00:00:00:00:00:00:02:1"
+            },
+            "primary_path": [
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:3"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:02:3"}}
+            ]
+        }
+        response = requests.post(api_url, data=json.dumps(payload),
+                                 headers={'Content-type': 'application/json'})
+        data = response.json()
+        evc1 = data['circuit_id']
+
+        payload = {
+            "primary_path": []
+        }
+
+        # It sets a new circuit's primary_path
+        response = requests.patch(api_url + evc1, data=json.dumps(payload),
+                                  headers={'Content-type': 'application/json'})
+        assert response.status_code == 400
+
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['primary_path'] != []
+        assert data['active'] is True
+
+    """It is returning Response [200], should be 400"""
+    @pytest.mark.xfail
+    def test_120_patch_an_inconsistent_backup_path(self):
+        api_url = KYTOS_API + '/mef_eline/v2/evc/'
+        payload1 = {
+            "name": "my evc1",
+            "enabled": True,
+            "uni_a": {
+                "interface_id": "00:00:00:00:00:00:00:01:1",
+            },
+            "uni_z": {
+                "interface_id": "00:00:00:00:00:00:00:02:1"
+            },
+            "backup_path": [
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:3"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:02:2"}}
+            ]
+        }
+        response = requests.post(api_url, data=json.dumps(payload1),
+                                 headers={'Content-type': 'application/json'})
+        data = response.json()
+        evc1 = data['circuit_id']
+
+        payload2 = {
+            "backup_path": [
+                {"endpoint_a": {"id": "00:00:00:00:00:00:00:01:1"},
+                 "endpoint_b": {"id": "00:00:00:00:00:00:00:02:1"}}
+            ]
+        }
+
+        # It sets a new circuit's primary_path
+        response = requests.patch(api_url + evc1, data=json.dumps(payload2),
+                                  headers={'Content-type': 'application/json'})
+        assert response.status_code == 400
+        time.sleep(10)
+
+        # It verifies EVC's data
+        response = requests.get(api_url + evc1)
+        data = response.json()
+        assert data['backup_path'][0]['endpoint_a']['id'] == payload1['backup_path'][0]['endpoint_a']['id']
+        assert data['backup_path'][0]['endpoint_b']['id'] == payload1['backup_path'][0]['endpoint_b']['id']
+        assert data['active'] is True
+
+    """It is returning Response [200], should be 400"""
+    @pytest.mark.xfail
+    def test_125_patch_an_unrelated_backup_path(self):
```

Comment:
Payload is missing the primary path

Reply:
This is resolved with commit [40c4a5f](https://github.com/amlight/kytos-end-to-end-tests/pull/69/commits/40c4a5f76aab55692bf67d74c0093c7958f2e6f6)